### PR TITLE
[IMP] accounting: new behavior of the sepa provider

### PR DESCRIPTION
**Pull request by Feyensv**
See https://github.com/odoo/odoo/commit/fd2fb212c50952ca5a8e162ba5d82ce433cd5989
See https://github.com/odoo/enterprise/commit/9b729b157b6978e10fbbed45052226f150e5559f

### CODE REVIEW

**Review by toaa-odoo — APPROVED** [review]
All good to me, maybe just change the wording of the part I commented to make it a bit clearer

File: content/applications/finance/accounting/payments/batch_sdd.rst [hunk 1]
```diff
@@ -50,17 +50,16 @@ SEPA Direct Debit as a payment method
...
 SDD can be used as a payment method both on your **eCommerce** or on the **Customer Portal** by
-activating SDD as a **Payment Provider**. With this method, your customers can create and sign their
-mandates themselves.
+activating SDD as a **Payment Provider**. With this method, your customers can create their mandates.
+To ensure the validity of the information given by the customer, they will have to confirm each
+mandate with one successful bank transfer of the expected amount.
 
-To do so, go to :menuselection:`Accounting --> Configuration --> Payment Providers`, click on *SEPA
-Direct Debit*, and set it up according to your needs.
 To do so, go to :menuselection:`Accounting app --> Configuration --> Payment Acquirers`, click on
```
Comment (AntoineVDV) [inline content/applications/finance/accounting/payments/batch_sdd.rst:57]:
```suggestion
To do so, go to :menuselection:`Accounting app --> Configuration --> Payment Providers`, click on
```

**Review by AntoineVDV — APPROVED** [review]
`applications/finance/payment_providers.rst#bank-payments` -> It shouldn't mention "signing" a mandate anymore.

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**